### PR TITLE
New package: DaCosySheeep.FactDownloader version 1.0

### PR DESCRIPTION
**Pull request by DaCosySheeep**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/128145)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @DaCosySheeep,

Manual Validation ended with:
image

Is there an installer switch to determine the install location? Many manifests split these into `user` and `machine` scopes, so users can choose.

**Comment by DaCosySheeep** [comment]
> Hi @DaCosySheeep,
> 
> Manual Validation ended with:
> image
> 
> Is there an installer switch to determine the install location? Many manifests split these into `user` and `machine` scopes, so users can choose. 

I used inno setup compiler. So I would have to search that up.

**Comment by DaCosySheeep** [comment]
How do I add a command line argument? /CURRENTUSER should do the trick.

File: manifests/d/DaCosySheeep/FactDownloader/1.0/DaCosySheeep.FactDownloader.locale.en-GB.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DaCosySheeep.FactDownloader
+PackageVersion: "1.0"
+PackageLocale: en-GB
+Publisher: DaCosySheeep
```
Comment (Trenly) [inline manifests/d/DaCosySheeep/FactDownloader/1.0/DaCosySheeep.FactDownloader.locale.en-GB.yaml:7]:
```suggestion
Publisher: Team Tobsta
```
This should match what is written in control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/d/DaCosySheeep/FactDownloader/1.0/DaCosySheeep.FactDownloader.locale.en-GB.yaml (hunk 1 above)
Comment (DaCosySheeep) [reply manifests/d/DaCosySheeep/FactDownloader/1.0/DaCosySheeep.FactDownloader.locale.en-GB.yaml:7]: It should be DaCosySheeep in the control panel. At least it is on the future releases. Team Tobsta is the name of my discord server LOL.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
When launched from File Explorer, this application launches normally. But when launched from command line, from another directory, it gives this message:
image

**Comment by DaCosySheeep** [comment]
> When launched from File Explorer, this application launches normally. But when launched from command line, from another directory, it gives this message: image

Ideally the user is launching it from the start menu. It should work fine from there. I can't imagine any of the users trying to launch it from command line.

**Comment by DaCosySheeep** [comment]
image
When will the package become available?

**Pull request merged**